### ISSUE TITLE
Heads up to everyone running the newist java and spigot 1.8.8

### ISSUE THREAD

**Issue by JBluesword** [Steffion/BlockHunt#358]
The plugin will lag your whole server and use 100% of your cpu. If u want to contine to use the plugin you will have to downgrade on java.

**Comment by SyraxR** [comment]
I agree! My server keeps making the commands have 3.48 seconds  of delay before executing!
By that i mean, 

> @JBluesword
